homeassistant/components/simplisafe/button.py

```diff
@@ -0,0 +1,93 @@
+"""Buttons for the SimpliSafe integration."""
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+
+from simplipy.errors import SimplipyError
+from simplipy.system import System
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SimpliSafe, SimpliSafeEntity
+from .const import DOMAIN
+from .typing import SystemType
+
+
+@dataclass
+class SimpliSafeButtonDescriptionMixin:
+    """Define an entity description mixin for SimpliSafe buttons."""
+
+    push_action: Callable[[System], Awaitable]
+
+
+@dataclass
+class SimpliSafeButtonDescription(
+    ButtonEntityDescription, SimpliSafeButtonDescriptionMixin
+):
+    """Describe a SimpliSafe button description."""
```

Comment:
Most other Button descriptions have the doc comment "Class to describe a XXX button entity" or even just "Class to describe a Button entity"

Reply:
Not sure what relevance that has here? This style mirrors the declarative style used throughout the SimpliSafe integration.

Reply:
I think that the comment should be 

```
"""Describe a SimpliSafe button entity"""
```

It describes an entity, not a description.  Just redundant.